user_docs/en/changes.md

```diff
@@ -5,6 +5,8 @@
 ### Important notes
 
 ### New Features
+* The volume of the other applications can be adjusted by `NVDA+alt+pageUp` and `NVDA+alt+pageDown`. (#16052, @mltony)
+* The sound of the other applications can be muted with `NVDA+alt+delete`. (#16052, @mltony)
```

Comment:
"The" not required in "The volume of other applications"

```suggestion
* The sound of other applications can be muted with `NVDA+alt+delete`. (#16052, @mltony)
```

Reply:
@Qchristensen, my bad: I am the one who has asked Tony to add "the".

My reasonning was that:
* "the other applications" was meaning "all the other applications, except NVDA"
* "other applications" may mean "some other applications but not necessarily all of them"

Given your comment and suggestion, I understand that it is not correct English.

My question:
If we write:
> * The sound of other applications can be muted with `NVDA+alt+delete`. (#16052, @mltony)

Do we understand that we are talking of the sound of all applications (implied except NVDa)?
Wouldn't be understood as "some of the other applications"?

If there is no confusion possible in English, go ahead with this wording.